binderhub/index.html

```diff
@@ -86,31 +88,34 @@
-                        to a notebook file, the notebook will be opened in your browser after building.
+                        <span class="front-em">Enter your repository information</span><br />Provide in the above form a URL or a GitHub repository that contains Jupyter notebooks, as well as a branch, tag, or commit hash. Launch will build your Binder repository. If you specify a path to a notebook file, the notebook will be opened in your browser after building.
                     </div>
                 </div>
 
                 <div class="row">
                     <div class="col-md-1 col-md-offset-2 point-container">
-                        <span class="point" style="border-color: rgb(208, 102, 129)">2</span>
+                        <span class="point point-red">2</span>
                     </div>
                     <div class="col-md-7 front">
-                        <span class="front-em">Let us build a Docker image of your repository.</span><br />Does your repository depend on files being loaded for your live notebooks? We will search in your repository's root for a dependency such as a Dockerfile or requirements.txt. The first dependency file found will be used to build a Docker image. If an image has already been built for the given repository, it will not be rebuilt. If a new commit has been made, the image will automatically be rebuilt.
+                        <span class="front-em">We build a Docker image of your repository</span><br />Does your repository depend on files being loaded for your live notebooks? We will search in your repository's root for a dependency such as a Dockerfile or requirements.txt. The first dependency file found will be used to build a Docker image. If an image has already been built for the given repository, it will not be rebuilt. If a new commit has been made, the image will automatically be rebuilt.
                     </div>
                 </div>
 
                 <div class="row">
                     <div class="col-md-1 col-md-offset-2 point-container">
-                        <span class="point" style="border-color: rgb(87, 154, 202)">3</span>
+                        <span class="point point-blue">3</span>
                     </div>
                     <div class="col-md-7 front">
                         <span class="front-em">Interact with your notebooks in a live environment!</span><br />A <a href="https://jupyterhub.readthedocs.io/en/latest/">JupyterHub</a> server will host your repository's contents. We offer you a reusable link and badge to your live repository that you can easily share with others.
                     </div>
                 </div>
+                <div class="row">
+                    <h3>questions? join the <a href="http://gitter.im/binder">chat</a>, see the <a href="https://github.com/jupyterhub/binderhub">code</a></h3>  
```

Comment:
I think this gitter isn't correct, it 404s for me...should be `https://gitter.im/binder-project/binder` ya?